### ISSUE TITLE
Incorporating Browser into other Unity Projects

### ISSUE THREAD

**Issue by JimothyMarmalade** [IanPhilips/UnityOculusAndroidVRBrowser#48]
I was just wondering if there are any more specific instructions that can be given on bringing the browser (and all necessary code/plugins) into another unity project. I've replicated the Underlay browser scene perfectly in another project but the application consistently crashes after a few frames after launching.

**Comment by JimothyMarmalade** [comment]
I figured out the problem.
The browser plugin -- GeckoView, specifically -- was consistently crashing because it was an ARMv7 build, and I was attempting to build for ARM64 unity project. I had set up the unity scene correctly, but not the proper Project settings to facilitate the issue.

I was able to get around this issue fairly easily by deleting the GeckoView ARMv7 plugin from the project files and replacing it with the ARM64 plugin **marked with the same version number.**

At the time of writing, the correct file to use is "geckoview-arm64-v8a-80.0.20200818235255.aar" from [the geckoview download page.](https://maven.mozilla.org/maven2/?prefix=maven2/org/mozilla/geckoview/geckoview-arm64-v8a/80.0.20200818235255/)
